mmpose/models/backbones/tcn.py

```diff
@@ -35,6 +37,7 @@ def __init__(self,
                  dropout=0.25,
                  causal=False,
                  residual=True,
+                 optimized_1f=False,
```

Comment:
The argument name and descriptions in the docstring can be more specific. For example, "train_with_stride_conv", would better help the user understand what this argument does.

Reply:
OK, I'll change that